packages/styled-base/types/index.d.ts

```diff
@@ -1,5 +1,5 @@
 // Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
-// TypeScript Version: 3.1
+// TypeScript Version: 3.2
```

Comment:
why the bump?

Reply:
Without it I get this:

Error: /Users/jakeg/_code/emotion/packages/styled-base/types/tests.tsx:332:18
ERROR: 332:18  expect  Compile error in typescript@3.1 but not in typescript@3.2.
Fix with a comment '// TypeScript Version: 3.2' just under the header.
Argument of type 'typeof A' is not assignable to parameter of type '"symbol" | "object" | "a" | "abbr" | "address" | "area" | "article" | "aside" | "audio" | "b" | "base" | "bdi" | "bdo" | "big" | "blockquote" | "body" | "br" | "button" | "canvas" | ... 154 more ... | "view"'.
  Type 'typeof A' is not assignable to type '"view"'.
ERROR: 355:18  expect  Compile error in typescript@3.1 but not in typescript@3.2.
Fix with a comment '// TypeScript Version: 3.2' just under the header.
Argument of type 'typeof C' is not assignable to parameter of type '"symbol" | "object" | "a" | "abbr" | "address" | "area" | "article" | "aside" | "audio" | "b" | "base" | "bdi" | "bdo" | "big" | "blockquote" | "body" | "br" | "button" | "canvas" | ... 154 more ... | "view"'.
  Type 'typeof C' is not assignable to type '"view"'.
ERROR: 361:1   expect  Expected an error on this line, but found none.